solr/core/src/java/org/apache/solr/core/SolrResourceLoader.java

```diff
@@ -697,7 +697,8 @@ void initCore(SolrCore core) {
       @Override
       protected void doReloadAction(Ctx ctx) {
         log.info("Core reloading classloader issued reload for: {}/{} ", coreName, coreId);
-        coreProvider.reload();
+        //do not reload core automatically. This is disabled for Fullstory
```

Comment:
Maybe also update the info log line above to indicate that we are skipping the core reload?